### ISSUE TITLE
Error while translating: "Missing variable (s) @count"

### ISSUE THREAD

**Issue by jenlampton** [backdrop-ops/localization#43]
Hello. While my friend was translating, I ran into an error here. The fault
is not on me. 

https://localize.backdropcms.org/translate/languages/tr/translate?project=backdropcms&status=1&release=42&search=&author&context=all&limit=50&sid=0
The error I get is this.

"Error message Missing variable (s) @count, @number_applied, @number_incompatible
• Missing variable (s) @count "

Even if you do a single random translation other than where he said it, you will know what I mean.

**Comment by olafgrabienski** [comment]
@jenlampton Are you reporting the issue for another person? However, I'm asking myself if it could be related to a warning message reported in https://github.com/backdrop-ops/localize.backdropcms.org/issues/38.

**Comment by jenlampton** [comment]
Yes, I was reporting for Mustafa Doğan since the report came in via the info@ email address.
